src/main/java/com/tournament/cup/settings/FirstClass.java

```diff
@@ -0,0 +1,56 @@
+package com.tournament.cup.settings;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.springframework.stereotype.Service;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+@Service
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class FirstClass {
+    TournamentClass tournamentClass;
+    AuthorizationClass authorizationClass;
+
+    public void mainScreen() {
+        System.out.println("\n                                                   CHAMPIONS LEAGUE");
+        System.out.println("                                                  PLAY IF YOU WORTHY     ");
+        System.out.print("\nAre you sure you want to play?\n");
+        System.out.print("Yes or No : ");
+        Scanner scanner = new Scanner(System.in);
+        String a = scanner.nextLine();
+        if (a.matches("Yes") || a.matches("yes")) {
```

Comment:
Always compare static values vs dynamic.
When `a` will be `null` you will not get NPE.